sql-constants.md

```diff
@@ -0,0 +1,220 @@
+---
+title: SQL Constant values
```

Comment:
Can we title this just `Constants`, like we used `Value Expressions` instead of `SQL Value Expressions`? The `SQL` is implied here.